tester/tester.go

```diff
@@ -383,6 +383,26 @@ func (tt *Tester) waitForClients() {
 	logger.Printf("waiting for consumers done")
 }
 
+// GetTableKeys returns a Table's keys.
+func (tt *Tester) GetTableKeys(table goka.Table) []string {
+	tt.mStorages.Lock()
+	defer tt.mStorages.Unlock()
+
+	keys := make([]string, 0)
+	topic := string(table)
+	st, exists := tt.storages[topic]
+	if !exists {
+		panic(fmt.Errorf("topic %s does not exist", topic))
+	}
+
+	it, _ := st.Iterator()
```

Comment:
`defer it.Release()`  is missing. I know it's just a tester, but who knows how our implementation might change in future and we require the iterators to be released...?

Reply:
Fair point.